client/src/app/site/settings/settings-list/settings-list.component.ts

```diff
@@ -19,7 +20,11 @@ export class SettingsListComponent extends BaseComponent implements OnInit {
      * @param titleService
      * @param translate
      */
-    public constructor(titleService: Title, protected translate: TranslateService) {
+    public constructor(
+        titleService: Title,
```

Comment:
should potentially also get public (now, to ensure the declaration rules are consistent with the linter, even tho linter does not (yet) complain here)

Reply:
We should not add a modifier here, because titleService is not used in this Class except for `super()`.